src/Components/Facility/HospitalList.tsx

```diff
@@ -343,7 +347,23 @@ export const HospitalList = (props: any) => {
                 )}
               </div>
               <div className="h-full w-full grow">
-                <div className="h-full flex flex-col justify-between w-full">
+                {isSmallScreen ? (
+                  <div className="group flex w-full self-stretch shrink-0 bg-gray-300 items-center justify-center relative z-0">
+                    {(facility.read_cover_image_url && (
+                      <img
+                        src={facility.read_cover_image_url}
+                        alt={facility.name}
+                        className="w-full h-full object-cover"
+                      />
+                    )) || (
+                      <i className="fas fa-hospital text-4xl block text-gray-500 p-10" />
+                    )}
+                  </div>
+                ) : (
+                  " "
+                )}
```

Comment:
```suggestion
                </div>
```

Reply:
The final result:

![image](https://user-images.githubusercontent.com/3626859/199274403-87c75a0c-128c-4b67-96b8-58975e655ee1.png)